test/nodes/test_llm_anthropic.py

```
@@ -0,0 +1,343 @@
+        ig = self._make_iglobal()
+        result = ig._format_error(None, None, None, 'Something went wrong')
+        assert result == 'Something went wrong'
+
+    def test_format_collapses_whitespace(self):
+        """Should collapse multiple spaces and strip."""
+        ig = self._make_iglobal()
+        result = ig._format_error(None, None, None, '  Too   many   spaces  ')
+        assert result == 'Too many spaces'
+
+    @pytest.mark.parametrize(
+        'status,etype,emsg,expected_fragment',
```

Ruff (0.15.7)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)